src/components/WalletConnect.vue

```diff
@@ -50,6 +51,25 @@ export default defineComponent({
 			WalletType,
 		};
 	},
+	mounted() {
+		searchForAssets("gov-token")
+			.then((response) => {
+				if (response && response.assets && response.assets.length) {
+					let assetLength: number = response.assets.length;
+					if (response.assets[assetLength - 1].params) {
+						console.log(
+							"Gov Token Info:",
+							response.assets[assetLength - 1].params
+						);
+					}
+				} else {
+					console.warn("gov-token not found");
```

Comment:
just to confirm, from a UI perspective -- we need to "display" this warning somehow. Right?

Reply:
Yes Right